flang/test/Driver/print-resource-dir.F90

```diff
@@ -0,0 +1,3 @@
+! RUN: %flang -print-resource-dir -resource-dir=%S/Inputs/resource_dir \
+! RUN:  | FileCheck -check-prefix=PRINT-RESOURCE-DIR -DFILE=%S/Inputs/resource_dir %s
+! PRINT-RESOURCE-DIR: [[FILE]]
```

Comment:
I have a suspicion that this would also print clang-specific stuff?

Reply:
It does not.  The `/Inputs/resource_dir` part has no relevance and I have copied that verbatim from what Clang does for its driver test.  I will change this to use DEFINE to simplify it.